libs/blocks/section-metadata/section-metadata.css

```diff
@@ -17,6 +17,10 @@
   color: rgb(34 34 34);
 }
 
+.section.static-links a:not([class]) {
```

Comment:
Interesting, I was thinking this as block level variant and only scoping it too the icon stack area but I hadn't considered the usage as a section style. 

Was there a discussion about this as a section style, or do we want to move it to block level?
ex.
```
.aside.static-links .icon-stack-area a {
    color: inherit;
    text-decoration: underline;
}

Reply:
@ryanmparrish There was a discussion in the "Milo engineering show and tell" meeting to add this style as a new style of section metadata block that would inherit the text color for the plain text links for the section. 
Also there was a similar requirement requested as a generic solution https://jira.corp.adobe.com/browse/MWPW-134186 and the change would help provide the functionality as a generic solution for a section.